### PR TITLE
feat: update openmldbapi exceptions class

### PR DESCRIPTION
**Pull request by HuilinWu2**
* **Please check if the PR fulfills these requirements**

- The commit message follows our guidelines
- Tests for the changes have been added (for bug fixes / features)
- Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

1. 【Openmldb Python-SDK】update the exceptions of  according to PEP 249 -- Python Database API Specification v2.0
2. add several code annotations

### CODE REVIEW

File: python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py [hunk 1]
```diff
@@ -17,10 +17,13 @@
 import logging
 from . import driver
 import re
+import exceptions
+
+# Globals
+apilevel = '2.0'  # String constant stating the supported DB API level.
```
Comment (tobegit3hub) [inline python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py]: Remove the comment which is duplicated with pep document.

Comment (HuilinWu2) [reply python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py]: updated

**Comment by HuilinWu2** [comment]
[PEP 249](https://www.python.org/dev/peps/pep-0249/) provides an example of Implementing Exceptions with importing exceptions.StandardError. However, this class is removed in Python3 and is replaced by "Exception" of _ _builtin_ _ module (no need to import).

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by vagetablechicken — APPROVED** [review]

**Comment by tobegit3hub** [comment]
[Make Python SDK available for users](https://trello.com/c/Mfqg2o9I/33-make-python-sdk-available-for-users)

**Pull request merged**